.cargo/config.toml

```diff
@@ -1,4 +1,4 @@
-[target.'cfg(all())']
+[build]
```

Comment:
With `[target.'cfg(all())']`, all allowed lints (such as `as` conversions, missing panics docs, etc.) were somehow still raising warnings.

I'm not sure if there are any cons of using the `build` table for this purpose, I'll raise this issue on EmbarkStudios/rust-ecosystem#59, will update this as required.